packages/visual-editor/src/hooks/useEditorSubscriber.ts

```diff
@@ -183,6 +183,8 @@ export function useEditorSubscriber() {
           break;
         }
         case INCOMING_EVENTS.DesignComponentsRegistered:
+          // Event was deprecated and support will be discontinued with version 5
+          break;
```

Comment:
Should we add explicit `console.warn(`event deprecatred ...yadayada`) ?  so that we can see if something is using it?

Reply:
I usually try to avoid swallowing errors as they eventually bite you where you don't expect.

But anyways this is merely a recommendation and feel free to act upon your own will :)

Reply:
Luckily, we already log each message in the console and only log additional warnings for unexpected ones. This one is expected to happen, so I would like to avoid polluting the logs with a separate log :)